plugins/git/git.plugin.zsh

```diff
@@ -65,6 +65,7 @@ alias gbss='git bisect start'
 
 alias gc='git commit -v'
 alias gc!='git commit -v --amend'
+alias gc-='git checkout -'
```

Comment:
Just out of curiosity, will this be an issue since we already have `gc` for `git commit`? 

Also, we also have an alias for `git checkout` as `gco`.

Reply:
There is no issue git existing `gc` alias. I've tested it.
This is one of my alias that I use always. Just wanted to share that it might be liked and used by others.